src/model/Position.ts

```diff
@@ -0,0 +1,17 @@
+import BN from "bn.js";
+
+export class Position {
+  liquidity: BN = new BN(0);
+  feeGrowthInside0LastX128: BN = new BN(0);
+  feeGrowthInside1LastX128: BN = new BN(0);
+  tokensOwed0: BN = new BN(0);
+  tokensOwed1: BN = new BN(0);
+
+  update(
```

Comment:
`update` mutates the internal data of the data structure, yet the fields that are being updated are also writable directly e.g. `position.feeGrowthInside0LastX128 = <some new value>`, this can cause inconsistency (updated via `update` v.s. updated directly), consider making the fields that are updated by `update` private, then make them read-only by adding getters.

Reply:
ok